metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -178,7 +180,7 @@ def create_job(
         job = (
             KubernetesClient()
             .job(
-                generate_name="t-",
+                generate_name="t-{uid}".format(uid=str(uuid4())[:3]),
```

Comment:
good catch with the python3.5 support. To echo the other comments, three characters won't be sufficient entropy for some cases. Proposing to pick the whole first section of the uuid instead so we're on the safe side:

```python
generate_name="t-{uid}-".format(uid=str(uuid4())[:8]),
```
(note the trailing dash as it's not appended automatically). First 8 chars should be sufficiently random for the purpose based on the V4 implementation, and this will still keep the generated pod name length reasonable.

Reply:
Was able to push the change to the branch, assuming no objections to the 8char length as the original proposal was even longer.

I'll go forward with this once tests pass so we don't keep this dangling any longer.